ecosystem/python/sdk/aptos_sdk/account_address.py

```diff
@@ -167,3 +210,97 @@ def test_token(self):
             base_address, "bob's collection", "bob's token"
         )
         self.assertEqual(actual, expected)
+
+    def test_to_standard_string(self):
+        # Test special address: 0x0
+        self.assertEqual(
+            AccountAddress.from_hex(
+                "0x0000000000000000000000000000000000000000000000000000000000000000"
+            ).to_standard_string(),
+            "0x0",
+        )
+
+        # Test special address: 0x1
+        self.assertEqual(
```

Comment:
> In other words, an address is special if the first 31 bytes are zero and the last byte is smaller than than `0b10000` (16)

does it mean that `0x0000000000000000000000000000000000000000000000000000000000000015` would become `0x15`? 
and `0x0000000000000000000000000000000000000000000000000000000000000016` would be `0x0000000000000000000000000000000000000000000000000000000000000016`?

Reply:
I believe the special addresses are just `0x0` to `0xf`

Anything that's a single hex char

Reply:
That's right ^